webapp/channels/src/components/threading/virtualized_thread_viewer/virtualized_thread_viewer.tsx

```diff
@@ -187,11 +187,10 @@ class ThreadViewerVirtualized extends PureComponent<Props, State> {
         if (scrollHeight <= 0) {
             return;
         }
-        const {createCommentHeight} = this.state;
 
         const updatedState: Partial<State> = {};
 
-        const userScrolledToBottom = scrollHeight - scrollOffset - createCommentHeight <= clientHeight;
+        const userScrolledToBottom = scrollHeight - scrollOffset - 5 <= clientHeight;
```

Comment:
@hmhealey I am not sure if this is the right solution but yeah it does fix the issue. I suspect it is caused by the new message separator we add on new message event. I have remove createCommentHeight, its outside virtualised list now

Reply:
If it's not the new message separator, I think it's fine to have a bit of leeway if the user isn't quite scrolled to the bottom. It'll also help account for any accidental layout shift that happens in the post list too. 

Could you make this number into a named constant to avoid the magic number? It can just live at the top of this file, but the name will help us remember what it's supposed to do later

Reply:
@hmhealey I have added a comment as well to make it clear why we have that constant do let me know if we need to make any changes.